### DISCUSSION TITLE
Release latest docker image

### DISCUSSION THREAD

**Discussion by lbennett-stacki** [strimzi/strimzi-kafka-operator#5391] [General]
Hello, latest docker image is 0.20, could we get 0.24?

**Comment by scholzj** [comment]
Due to the usage limits, we moved from Docker Hub to Quay.io some time ago. The images are now here: https://quay.io/organization/strimzi ... is it that you are looking for Docker Hub?

**Reply by lbennett-stacki** [reply]
Aha! Thanks, I had a feeling something like this could have happened. Yeah, I was updating some kafka connect docker images to use now secrets/configmaps config classes and using the dockerhub images.

**Reply by scholzj** [reply]
Did you found the link to the Docker Hub link in some of our recent docs or README files? If yes, we should update it. If it was google or something like that, there is probably not much we can do about it.
